gh/gh.go

```diff
@@ -57,7 +58,7 @@ func (g *GHub) RemoveLabels(issue *github.Issue, repoID types.RepoID, labelsToRe
 		return err
 	}
 
-	newLabels := []string{}
+	var newLabels []string
```

Comment:
That should not be changed, because if we don't append any labels we send nil array to the github api and receives an error.
```

2018/02/15 11:29:19 PUT https://api.github.com/repos/Soluto-Private/anywhere-expert-app/issues/337/labels: 422 Invalid request.

For 'links/3/schema', nil is not an array. []
```

Reply:
pb fixed.